src/input/Input.ts

```diff
@@ -24,24 +47,68 @@ export class Input extends EventEmitter<{
     pointerupoutside: InputEvent;
     pointerdown: InputEvent;
     pointertap: InputEvent;
+
+    wheel: InputEvent;
+    globalwheel: InputEvent;
 }>
 {
     /**
      * Determines if the children to the container can be clicked/touched
      * Setting this to false allows PixiJS to bypass a recursive `hitTest` function
      */
     public interactiveChildren = true;
-    public _globalMove = false;
     /** The cursor preferred when the mouse pointer is hovering over. */
     public cursor: Cursor | string = 'default';
+    /** Handler for 'pointermove' event */
+    public onpointermove: (i: InputEvent) => void;
+    /** Handler for 'globalpointermove' event */
+    public onglobalpointermove: (i: InputEvent) => void;
+    /** Handler for 'pointerover' event */
+    public onpointerover: (i: InputEvent) => void;
+    /** Handler for 'pointerout' event */
+    public onpointerout: (i: InputEvent) => void;
+    /** Handler for 'pointerup' event */
+    public onpointerup: (i: InputEvent) => void;
+    /** Handler for 'pointerupoutside' event */
+    public onpointerupoutside: (i: InputEvent) => void;
+    /** Handler for 'pointerdown' event */
+    public onpointerdown: (i: InputEvent) => void;
+    /** Handler for 'pointertap' event */
+    public onpointertap: (i: InputEvent) => void;
+
+    /** Handler for 'wheel' event */
+    public onwheel: (i: InputEvent) => void;
+    /** Handler for 'globalwheel' event */
+    public onglobalwheel: (i: InputEvent) => void;
+
+    public _globalMove = false;
+    public _globalWheel = false;
     public _isPointer = false;
     protected container: Container = null;
-    protected _enable = false;
+    protected _interactive = false;
 
-    constructor(container: Container)
+    constructor(options: InputOptions)
     {
         super();
-        this.container = container;
+        this.container = options.target;
+        this.interactive = options.interactive ?? false;
+        this.globalMove = options.globalMove ?? false;
+        this.cursor = options.cursor ?? 'default';
+        this.isPointer = options.isPointer ?? false;
+        this.interactiveChildren = options.interactiveChildren ?? true;
+
+        this.onpointermove ??= options.onpointermove;
+        this.onglobalpointermove ??= options.onglobalpointermove;
+        this.onpointerover ??= options.onpointerover;
+        this.onpointerout ??= options.onpointerout;
+        this.onpointerup ??= options.onpointerup;
+        this.onpointerupoutside ??= options.onpointerupoutside;
+        this.onpointerdown ??= options.onpointerdown;
+        this.onpointertap ??= options.onpointertap;
+
+        this.globalWheel = options.globalWheel ?? false;
+        this.onwheel ??= options.onwheel;
+        this.onglobalwheel ??= options.onglobalwheel;
```

Comment:
Why not use assign here? It's better pattern for defaults. because `target` is a different name the arg should probably be `{ target, ...options }: Options`

```suggestion
        this.container = target;
        Object.assign(this, {
            interactive: false,
            globalMove: false,
            cursor: 'default',
            isPointer: false,
            interactiveChildren: true,
            globalWheel: false,
        }, options);
```